.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.singleQuote": true
```

Comment:
you can also customize it on the global settings of VScode, that will help you to avoid the creation of this file in each of your projects